roles/rsyslog/defaults/main.yml

```diff
@@ -178,9 +166,9 @@ __rsyslog_conf_global_options:
       global(
         defaultNetstreamDriver="{{ rsyslog_default_netstream_driver }}"
         workDirectory="{{ rsyslog_work_dir }}"
-      {% if rsyslog_pki | bool and "tls" in rsyslog_capabilities %}
+      {% if __rsyslog_encryption_map[logging_encryption] | d("ptcp") != "ptcp" %}
```

Comment:
can you use `rsyslog_default_netstream_driver` here instead of `__rsyslog_encryption_map[logging_encryption]` ?

Reply:
You are right, @richm. I should have used the variable instead of the dict... :p
```
./roles/rsyslog/vars/outputs/forwards/main.yml:    state: '{{ "present" if __rsyslog_encryption_map[logging_encryption] | d("ptcp") != "ptcp"
./roles/rsyslog/tasks/main.yml:           {{ __rsyslog_tls_packages if __rsyslog_encryption_map[logging_encryption] | d('ptcp') != 'ptcp' else [] }} +
./roles/rsyslog/defaults/main.yml:      {% if __rsyslog_encryption_map[logging_encryption] | d("ptcp") != "ptcp" %}
```